### PR TITLE
[stable/aws-alb-ingress-controller] Adding priority class support 🔺

### PR DESCRIPTION
**Pull request by johndaviesco**
Summoning @bigkraig and/or @M00nF1sh  per the PR template. Thanks for taking the time to review this, it's much appreciated!

####Special notes for your reviewer:
I installed this change on an EKS cluster and the  priority class appeared as expected.

#### What this PR does / why we need it:
Adding `priorityClassName` support for aws-alb-ingress-controller.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by johndaviesco** [comment]
Hey @komljen could you run the `ok-to-test` on this please. @M00nF1sh tried but didn't have permissions

**Comment by komljen** [comment]
I can try, but it will not work as I'm not in the owners list of this chart. @M00nF1sh Should be able to do it, not sure why it didn't work.

**Comment by komljen** [comment]
Ah, actually it worked :)

**Comment by johndaviesco** [comment]
@komljen could you also give it a `/lgtm`

**Comment by johndaviesco** [comment]
Hey @M00nF1sh your in the owners file do you want to try `/lgtm` if you approve ?

**Comment by johndaviesco** [comment]
Hey @bigkraig your in the owners file do you want to try /lgtm if you approve ?

File: incubator/aws-alb-ingress-controller/README.md
Comment (bigkraig) [inline incubator/aws-alb-ingress-controller/README.md]: can you add a description here of what the setting is?

File: incubator/aws-alb-ingress-controller/README.md
Comment (bigkraig) [inline incubator/aws-alb-ingress-controller/README.md]: it looks like this line was corrupted

**Review by bigkraig — CHANGES_REQUESTED** [review]

**Comment by johndaviesco** [comment]
Hey @bigkraig I think I have addressed your concerns can you please review ?

**Pull request merged**
